### ISSUE TITLE
0.11.0: documentation build fails

### ISSUE THREAD

**Issue by kloczek** [jupyterlite/jupyterlite-sphinx#117]
Looks like `jupyterlite-sphinx documentation build fails:
```console
+ /usr/bin/sphinx-build -n -T -b man docs build/sphinx/man
Running Sphinx v7.1.2
myst v2.0.0: MdParserConfig(commonmark_only=False, gfm_only=False, enable_extensions=set(), disable_syntax=[], all_links_external=False, url_schemes=('http', 'https', 'mailto', 'ftp'), ref_domains=None, fence_as_directive=set(), number_code_blocks=[], title_to_header=False, heading_anchors=0, heading_slug_func=None, html_meta={}, footnote_transition=True, words_per_minute=200, substitutions={}, linkify_fuzzy_links=True, dmath_allow_labels=True, dmath_allow_space=True, dmath_allow_digits=True, dmath_double_inline=False, update_mathjax=True, mathjax_classes='tex2jax_process|mathjax_process|math|output_area', enable_checkboxes=False, suppress_warnings=[], highlight_code_blocks=True)
building [mo]: targets for 0 po files that are out of date
writing output...
building [man]: all manpages
updating environment: [new config] 10 added, 0 changed, 0 removed
reading sources... [100%] installation
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
writing... python-jupyterlite-sphinx.3 { failed

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/sphinx/cmd/build.py", line 290, in build_main
    app.build(args.force_all, args.filenames)
  File "/usr/lib/python3.8/site-packages/sphinx/application.py", line 351, in build
    self.builder.build_update()
  File "/usr/lib/python3.8/site-packages/sphinx/builders/__init__.py", line 287, in build_update
    self.build(['__all__'], to_build)
  File "/usr/lib/python3.8/site-packages/sphinx/builders/__init__.py", line 360, in build
    self.write(docnames, list(updated_docnames), method)
  File "/usr/lib/python3.8/site-packages/sphinx/util/display.py", line 91, in wrapper
    return f(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/sphinx/builders/manpage.py", line 92, in write
    largetree = inline_all_toctrees(self, docnames, docname, tree,
  File "/usr/lib/python3.8/site-packages/sphinx/util/nodes.py", line 406, in inline_all_toctrees
    tree = tree.deepcopy()
  File "/usr/lib/python3.8/site-packages/docutils/nodes.py", line 1065, in deepcopy
    copy.extend([child.deepcopy() for child in self.children])
  File "/usr/lib/python3.8/site-packages/docutils/nodes.py", line 1065, in <listcomp>
    copy.extend([child.deepcopy() for child in self.children])
  File "/usr/lib/python3.8/site-packages/docutils/nodes.py", line 1065, in deepcopy
    copy.extend([child.deepcopy() for child in self.children])
  File "/usr/lib/python3.8/site-packages/docutils/nodes.py", line 1065, in <listcomp>
    copy.extend([child.deepcopy() for child in self.children])
  File "/usr/lib/python3.8/site-packages/docutils/nodes.py", line 1064, in deepcopy
    copy = self.copy()
  File "/usr/lib/python3.8/site-packages/sphinx/util/nodes.py", line 627, in _copy_except__document
    newnode = self.__class__(rawsource=self.rawsource, **self.attributes)
  File "/home/tkloczko/rpmbuild/BUILD/jupyterlite-sphinx-0.11.0/jupyterlite_sphinx/jupyterlite_sphinx.py", line 142, in __init__
    super().__init__(rawsource, *children, iframe_src=iframe_src, **attributes)
TypeError: __init__() got multiple values for keyword argument 'iframe_src'

Exception occurred:
  File "/home/tkloczko/rpmbuild/BUILD/jupyterlite-sphinx-0.11.0/jupyterlite_sphinx/jupyterlite_sphinx.py", line 142, in __init__
    super().__init__(rawsource, *children, iframe_src=iframe_src, **attributes)
TypeError: __init__() got multiple values for keyword argument 'iframe_src'
The full traceback has been saved in /tmp/sphinx-err-6xq42lnr.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

**Comment by martinRenou** [comment]
This is surprising, the docs build fine on readthedocs and I just succeeded building locally too.

Could you provide steps to reproduce this? Do you have a custom `conf.py`? Also could you tell us which packages versions you have?

**Comment by kloczek** [comment]
Did you try to build man page? 🤔

**Comment by martinRenou** [comment]
Yes

**Comment by kloczek** [comment]
Only patch which I'm using is below one
```patch
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -1,3 +1,7 @@
+import sys
+import os
+sys.path.insert(0, os.path.abspath("../src"))
+
 # -*- coding: utf-8 -*-

 extensions = [
```
to allow build documentation without have `jupyterlite-sphinx` installed.

**Comment by Carreau** [comment]
I can reproduce with sphinx 7.1.2, and it seem to be fixed in 7.2.0

I've tracked that down to 6f5a99a05d0f760cebadcb15f55a952fcff02bc2, https://github.com/sphinx-doc/sphinx/pull/11565.

I'll send a PR with a better error message. I guess we could pin sphinx, but as this affects only man generation I'm unsure it is necessary.
